go/vt/vtctl/vtctl.go

```diff
@@ -478,7 +478,7 @@ var commands = []commandGroup{
 		"Workflow", []command{
 			{"Workflow", commandWorkflow,
 				"<ks.workflow> <action> --dry-run",
-				"Start/Stop/Delete Workflow on all target tablets in workflow. Example: Workflow merchant.morders Start",
+				"Start/Stop/Delete/List/List-All Workflow on all target tablets in workflow. Example: Workflow merchant.morders Start",
```

Comment:
Some suggested renames:
`List`->`Show`
`List-All` -> `ListAll`

Reply:
@sougou List was already there, but just not documented here in the text. It feels a bit out of scope to rename that subcommand in this PR. Are you ok with renaming it in another PR. Also @rohit-nayak-ps thoughts on the naming?

Reply:
What do you think the analog for `ListAll` is in the command itself? just `listall` with no hyphen?

Reply:
@PrismaPhonic, I will work on the List rename and parameter ordering changes Sugu suggested as part of some upcoming changes in the list response (need to report blacklisted tables, routing rules related to the workflow, among other info ...)

Reply:
Renamed `List-All` to `ListAll` as our subcommand naming convention is `PascalCase`.